### PR TITLE
Change margin bottom to zero when the paragraph tag is last child

### LINKED ISSUE #62: Remove margin-bottom on p when it's the last child
Code can be found currently in `\content\Assets\Styles\base\_paragraph.scss`

### PR DESCRIPTION
**Pull request by Newnab**
Nice and simple! Fixes #62

### CODE REVIEW

File: content/Assets/Styles/base/_paragraph.scss [hunk 1]
```diff
@@ -9,4 +9,8 @@
 p {
     @include rem(margin-bottom, $spacing-default);
     margin-top: 0;
+
+    &:last-child{
+        margin-bottom:0;
```
Comment (peterkeating) [inline content/Assets/Styles/base/_paragraph.scss]: For consistency, can you include a space after the `:`.

Comment (mobinzk) [reply content/Assets/Styles/base/_paragraph.scss]: Also between `:last-child` and `{`.

Comment (mobinzk) [reply content/Assets/Styles/base/_paragraph.scss]: For reference I'm using this:
https://github.com/prettier/prettier-vscode.git

Which fixes the formatting on save.

Comment (Newnab) [reply content/Assets/Styles/base/_paragraph.scss]: Interestingly, I have that installed too. I wonder why it's not working, or if there's an option I need to configure...?

Comment (peterkeating) [reply content/Assets/Styles/base/_paragraph.scss]: I actually have `prettier` turned off for CSS myself. However, I'm wondering if we're missing some settings within the `.prettierrc` file thus your editors are implying their own configurations.

**Pull request merged**